zeek/shaper.zed

```diff
@@ -54,56 +54,6 @@ type weird={_path:string,ts:time,uid:bstring,id:id,name:bstring,addl:bstring,not
-    case _path=snmp => put . = shape(snmp)
-    case _path=ssh => put . = shape(ssh)
-    case _path=syslog => put . = shape(syslog)
-    case _path=x509 => put . = shape(x509)
-    case _path=capture_loss => put . = shape(capture_loss)
-    case _path=dce_rpc => put . = shape(dce_rpc)
-    case _path=ftp => put . = shape(ftp)
-    case _path=notice => put . = shape(notice)
-    case _path=packet_filter => put . = shape(packet_filter)
-    case _path=radius => put . = shape(radius)
-    case _path=smtp => put . = shape(smtp)
-    case _path=stats => put . = shape(stats)
-    case _path=weird => put . = shape(weird)
-    case _path=dpd => put . = shape(dpd)
-    case _path=intel => put . = shape(intel)
-    case _path=irc => put . = shape(irc)
-    case _path=kerberos => put . = shape(kerberos)
-    case _path=loaded_scripts => put . = shape(loaded_scripts)
-    case _path=mysql => put . = shape(mysql)
-    case _path=netcontrol => put . = shape(netcontrol)
-    case _path=netcontrol_shunt => put . = shape(netcontrol_shunt)
-    case _path=signatures => put . = shape(signatures)
-    case _path=socks => put . = shape(socks)
-    case _path=software => put . = shape(software)
-    case _path=known_services => put . = shape(known_services)
-    case _path=sip => put . = shape(sip)
-    case _path=smb_mapping => put . = shape(smb_mapping)
-    case _path=broker => put . = shape(broker)
-    case _path=config => put . = shape(config)
-    case _path=notice_alarm => put . = shape(notice_alarm)
-    case _path=pe => put . = shape(pe)
-    case _path=smb_files => put . = shape(smb_files)
-    case _path=cluster => put . = shape(cluster)
-    case _path=conn => put . = shape(conn)
-    case _path=dnp3 => put . = shape(dnp3)
-    case _path=known_hosts => put . = shape(known_hosts)
-    case _path=netcontrol_drop => put . = shape(netcontrol_drop)
-    case _path=ntlm => put . = shape(ntlm)
-    case _path=reporter => put . = shape(reporter)
-    case _path=rfb => put . = shape(rfb)
-)
+const schemas = |{ "broker": broker, "capture_loss": capture_loss, "cluster": cluster, "config": config, "conn": conn, "dce_rpc": dce_rpc, "dhcp": dhcp, "dnp3": dnp3, "dns": dns, "dpd": dpd, "files": files, "ftp": ftp, "http": http, "intel": intel, "irc": irc, "kerberos": kerberos, "known_certs": known_certs, "known_hosts": known_hosts, "known_services": known_services, "loaded_scripts": loaded_scripts, "modbus": modbus, "mysql": mysql, "netcontrol": netcontrol, "netcontrol_drop": netcontrol_drop, "netcontrol_shunt": netcontrol_shunt, "notice": notice, "notice_alarm": notice_alarm, "ntlm": ntlm, "ntp": ntp, "packet_filter": packet_filter, "pe": pe, "radius": radius, "rdp": rdp, "reporter": reporter, "rfb": rfb, "signatures": signatures, "sip": sip, "smb_files": smb_files, "smb_mapping": smb_mapping, "smtp": smtp, "snmp": snmp, "socks": socks, "software": software, "ssh": ssh, "ssl": ssl, "stats": stats, "syslog": syslog, "tunnel": tunnel, "weird": weird, "x509": x509 }|
```

Comment:
Maybe break this across multiple lines?

Reply:
Sure. I could have gone either way on that one. Part of me was geeked at the idea of cutting the line count of the shaper config by half, but I know super long lines present their own challenges, so I've broken it across multiple lines. :smile: